src/views/RealTimeEvents/TabsView.vue

```diff
@@ -34,6 +34,8 @@
   import { useRoute, useRouter } from 'vue-router'
   import TabView from 'primevue/tabview'
   import TabPanel from 'primevue/tabpanel'
+  import { GetRelevantField } from '@/modules/real-time-metrics/filters'
```

Comment:
Should we import a module from Metrics on Events? 🤔

Reply:
Esse `GetRelevantField` chama o helper de real-time-filters-rules que é para ambos os cenários de metrics e events, mas posso criar um novo modulo que seja somente para o real time events